importer_client/python/timesketch_import_client/importer.py

```diff
@@ -723,8 +715,14 @@ def close(self):
         except ValueError:
             return
 
-        if self._data_lines:
-            self.flush(end_stream=True)
+        # TODO(jaegeral): End of 11/2023 from now remove the code below
+        # if no new bugs occur
+        # Commenting out because of https://github.com/google/timesketch/issues/2796
+        # testing it didn't seem to cause any issues as if flush is called with
+        # end_stream=True it will just return so there is no reason
+        # to call it here.
+        # if self._data_lines:
+        #    self.flush(end_stream=True)
```

Comment:
Please add a test for this. Is this replicable with a test file of > 50k entries (the flush threashold)?

Reply:
removed it for now, I am introducing a test for < 50 k events now, if that is merged in, I will add another for < 50 k events